### ISSUE TITLE
Rewrite nova

### ISSUE THREAD

**Issue by basepi** [hubblestack/hubble#536]
Offload all of the processing code to the core audit module for DRY and consistency reasons. This will also allow us to drastically reduce the complexity of the profiles themselves.

**Comment by basepi** [comment]
https://jira.corp.adobe.com/browse/CSTTEAM-43970

**Comment by alias454** [comment]
Not sure if you are up for requests at this point but having a utility to convert existing audit files to the new format would be nice(whatever that may be).

**Comment by basepi** [comment]
Yep, I will definitely handle the conversion. I'll probably have to write a slightly different routine for each audit module, but that will be *way* easier than rewriting the profiles from scratch.

**Comment by alias454** [comment]
Cool. I wrote up something that will convert the yaml to csv then another script to read in the csv. The second stage uses pandas to read the csv and update a sqlite db. I'll attach the first stage script if it might be helpful in anyway. Feel free to use it or not. The one thing I didn't do was split up the list of checks. I decided to just stare that as a json blob. 
[nova_audit_csv_prep.txt](https://github.com/hubblestack/hubble/files/2805189/nova_audit_csv_prep.txt)
Also, don't laugh since I'm not a programmer ;)
